Source/Engine/Syntax/AnySpanSyntax.cs

```diff
@@ -3,12 +3,6 @@
 // Licensed under the Apache License, Version 2.0.
 //--------------------------------------------------------------------------------------------------
 
-using System;
```

Comment:
Please, don't remove `using`s that seem useless, they allow better code completion (using namespace don't create assembly dependency).